src/pages/[[...markdownPath]].js

```
@@ -6,10 +6,12 @@ import {Fragment, useMemo} from 'react';
 import {useRouter} from 'next/router';
 import {Page} from 'components/Layout/Page';
 import sidebarHome from '../sidebarHome.json';
-import sidebarLearn from '../sidebarLearn.json';
-import sidebarReference from '../sidebarReference.json';
-import sidebarCommunity from '../sidebarCommunity.json';
-import sidebarBlog from '../sidebarBlog.json';
+import sidebarHome from '../sidebarHome.json';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'sidebarHome'. Consider to delete it or rename it.

'sidebarHome' is defined here:


(lint/suspicious/noRedeclare)